src/ahbichtfunctionsclient/__init__.py

```diff
@@ -4,13 +4,19 @@
 """
 import aiohttp
 from ahbicht.expressions.package_expansion import PackageResolver
-from ahbicht.mapping_results import PackageKeyConditionExpressionMapping, PackageKeyConditionExpressionMappingSchema
+from ahbicht.mapping_results import (
+    ConditionKeyConditionTextMapping,
+    ConditionKeyConditionTextMappingSchema,
+    PackageKeyConditionExpressionMapping,
+    PackageKeyConditionExpressionMappingSchema,
+)
+from maus.edifact import EdifactFormat, EdifactFormatVersion
 
 
 # pylint: disable=too-few-public-methods
 class HochfrequenzPackageResolver(PackageResolver):
     """
-    A package resolver that uses a REST API (by Hochfrequenz) to resolve packages.
+    A package resolver that uses a REST API (by Hochfrequenz) to request a condition expression for a given package key.
```

Comment:
```suggestion
    A package resolver that uses a REST API (by Hochfrequenz) to retrieve a condition expression for a given package key.
```
für mich ein kleiner unterschied. 
"request" ist für mich die antwort nach dem "Wie?"
"retrieve" ist die Antwort auf "Was?"

IMHO sollte der Docstring in erster Linie das "was" beschreiben und erst danach das "wie".